docs/reference/index-modules.asciidoc

```diff
@@ -79,6 +79,12 @@ Checking shards may take a lot of time on large indices.
     which uses https://en.wikipedia.org/wiki/DEFLATE[DEFLATE] for a higher
     compression ratio, at the expense of slower stored fields performance.
 
+`index.routing_partition_size`::
```

Comment:
Add an ID for linking

    [[routing-partition-size]]
    `index.routing_partition_size`::